retrieverdash/dashboard_script/status_dashboard_tools.py

```diff
@@ -55,6 +60,13 @@ def create_diff(csv1, csv2, diff_file, context, numlines):
     csv1 : The first csv file.
     csv2 : The second csv file.
     diff_file : The diff_file that is to be generated.
+    context : set to True for contextual differences (defaults to False
+            which shows full differences).
```

Comment:
I feel the second part needs to be made more clear.
  `numlines : Number of context lines before and after diff.
    Set to True to controls number of code lines before and after the change.
    Set to False .....` I am having hard time to comprehend what `Set to False..` does

Reply:
Setting to false shows the whole file. The changes and also those lines which do not have changes.

Reply:
And when False is set for context the numlines are used for positioning the `next` anchor tag before the change. If context is set to false and numlines is set to 2 then it will place the anchor tag 2 lines before the change.